languages/omise-woocommerce-ja.po

```diff
@@ -0,0 +1,449 @@
+msgstr "振込可能な(%s)の一部を口座へ振込む"
+
+#: includes/templates/partials/transfer-box-template.php:27
+msgctxt "Transfer amount"
+msgid "Amount"
+msgstr "金額"
+
+#: includes/templates/partials/transfer-box-template.php:33
+msgid "Transfer"
+msgstr "振込ID"
+
+#: includes/templates/partials/transfer-box-template.php:34
+msgid "Cancel"
+msgstr "キャンセル"
+
+#: includes/templates/partials/transfer-box-template.php:39
+msgid ""
+"Transfers are processed daily in chronological order. If balance is "
+"insufficient, transfers are retried on the next day."
+msgstr ""
+"振込依頼は毎日リクエスト順に処理されます。残高不足の場合は翌日に再処理が行な"
+"われます"
+
+#: omise-wc-gateway.php:47
+msgid "Enable/Disable"
+msgstr "有効/無効"
+
+#: omise-wc-gateway.php:49
+msgid "Enable Omise Payment Module."
+msgstr "Omiseのモジュール有効"
+
+#: omise-wc-gateway.php:53
+msgid "Sandbox"
+msgstr "テスト環境"
+
+#: omise-wc-gateway.php:55
+msgid "Enabling sandbox means that all your transactions will be in TEST mode."
+msgstr "テスト環境を有効にすると全てテストで実行します"
+
+#: omise-wc-gateway.php:59
+msgid "Public key for test"
+msgstr "テスト用のパブリックキー"
+
+#: omise-wc-gateway.php:61
+msgid "The \"Test\" mode public key can be found in Omise Dashboard."
+msgstr "テスト用のパブリックキーはOmiseのテストダッシュボードの中にあります"
+
+#: omise-wc-gateway.php:64
+msgid "Secret key for test"
+msgstr "テスト用のシークレットキー"
+
+#: omise-wc-gateway.php:66
+msgid "The \"Test\" mode secret key can be found in Omise Dashboard."
+msgstr "テスト用のシークレットキーはOmiseのテストダッシュボードの中にあります"
+
+#: omise-wc-gateway.php:69
+msgid "Public key for live"
+msgstr "ライブ用のパブリックキー"
+
+#: omise-wc-gateway.php:71
+msgid "The \"Live\" mode public key can be found in Omise Dashboard."
+msgstr "ライブ用のパブリックキーはOmiseのライブダッシュボードの中にあります"
+
+#: omise-wc-gateway.php:74
+msgid "Secret key for live"
+msgstr "ライブ用のシークレットキー"
+
+#: omise-wc-gateway.php:76
+msgid "The \"Live\" mode secret key can be found in Omise Dashboard."
+msgstr "ライブ用のシークレットキーはOmiseのライブダッシュボードの中にあります"
+
+#: omise-wc-gateway.php:79
+msgid "Advance Settings"
+msgstr "詳細設定"
+
+#: omise-wc-gateway.php:84
+msgctxt "Setting title"
+msgid "Title"
+msgstr "タイトル"
+
+#: omise-wc-gateway.php:86
+msgid "This controls the title which the user sees during checkout."
+msgstr "チェックアウトページのタイトルを設定する"
+
+#: omise-wc-gateway.php:87
+msgctxt "Default title"
+msgid "Omise Payment Gateway"
+msgstr "Omise Payment Gateway"
+
+#: omise-wc-gateway.php:90
+msgid "Payment action"
+msgstr "売上方法を選択"
+
+#: omise-wc-gateway.php:92
+msgid "Manual Capture or Capture Automatically"
+msgstr "手動売上/自動売上"
+
+#: omise-wc-gateway.php:99
+msgid "3-D Secure support"
+msgstr "3-D Secure のサポート"
+
+#: omise-wc-gateway.php:101
+msgid ""
+"Enable or disable 3-D Secure for the account. (Japan-based accounts are not "
+"eligible for the service.)"
+msgstr "アカウントを3-D Secure 有効/無効にする（日本派にはご利用出来ません）"
+
+#: omise-wc-gateway.php:105
+msgctxt "Setting description"
+msgid "Description"
+msgstr "説明"
+
+#: omise-wc-gateway.php:107
+msgid "This controls the description which the user sees during checkout."
+msgstr "チェックアウトページの説明を設定する"
```

Comment:
+msgstr "チェックアウトページの説明を設定する"
👇 
+msgstr "This controls the description which the user sees during checkout."

Since it's not used and to be deleted, let's leave it as English